### PR TITLE
Fix latest dependency checker install command

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

File: .github/workflows/latest_dependency_checker.yaml [hunk 1]
```diff
@@ -22,7 +22,8 @@ jobs:
       - name: Update dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -e .[spark, test]
```
Comment (thehomebrewnerd) [inline .github/workflows/latest_dependency_checker.yaml:25]: Looks like the previous command was `python -m pip install .[spark,test]` without a space between. Would that work here? Might be nice to install them both together as the dependency resolver might arrive at different package versions when we do a separate install - I don't think pip takes into account already installed packages for subsequent install commands.

File: .github/workflows/latest_dependency_checker.yaml [hunk 2]
```diff
@@ -22,7 +22,8 @@ jobs:
...
+          python -m pip install -e ".[spark]"
```
Comment (gsheni) [inline .github/workflows/latest_dependency_checker.yaml:25]:
```suggestion
          python -m pip install -e ".[spark,test]"
```

File: .github/workflows/latest_dependency_checker.yaml (hunk 1 above)
Comment (gsheni) [reply .github/workflows/latest_dependency_checker.yaml:25]: Fixed

File: pyproject.toml [hunk 3]
```diff
@@ -104,13 +104,13 @@ docs = [
...
 ]
 dev = [
     "ruff >= 0.0.239",
     "black[jupyter] >= 23.1.0",
     "pre-commit >= 2.20.0",
-    "featuretools[docs, spark, test]",
+    "featuretools[docs,spark,test]",
 ]
 complete = [
     "featuretools[autonormalize, nlp, sklearn, spark, sql, tsfresh, updater]",
```
Comment (gsheni) [inline pyproject.toml:116]:
```suggestion
    "featuretools[autonormalize,nlp,sklearn,spark,sql,tsfresh,updater]",
```

**Review by thehomebrewnerd — APPROVED** [review]

**Pull request merged**